docs/reference/config.md

```diff
@@ -12,7 +12,10 @@ This page lists all of the available settings in the {ref}`Nextflow configuratio
 : If `true`, on a successful completion of a run all files in *work* directory are automatically deleted.
 
   :::{warning}
-  The use of the `cleanup` option will prevent the use of the *resume* feature on subsequent executions of that pipeline run. Also, be aware that deleting all scratch files can take a lot of time, especially when using a shared file system or remote cloud storage.
+  The use of the `cleanup` option will prevent the use of the *resume* feature on subsequent executions of that pipeline run.
+  :::
```

Comment:
Double `:::` crept in here. Also it's indented, which feels unusual?

Reply:
These are two warnings, one after the other so the double `:::` is expected. The indentation is also normal as it's listed within the reference list.

Reply:
Ok fab, thanks for checking!